packages/slider/CHANGELOG.md

```diff
@@ -11,10 +11,6 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 -   **slider:** allow handler to be focusable when slider is editable ([23f8b89](https://github.com/adobe/spectrum-web-components/commit/23f8b8908a288f246f6330292b7ff090f931822a))
```

Comment:
Also remove this if you remove the revert

Reply:
The revert section has been removed. May I know why do you want me to remove this line?

Reply:
My bad! I overlooked